test/functional/idb-e2e-specs.js

```diff
@@ -0,0 +1,47 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { getSimulator } from 'appium-ios-simulator';
```

Comment:
can we avoid using `appium-ios-simulator` module? otherwise there will be a circular dependency